qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -62,6 +67,9 @@
             image_name_stg = "images/storage"
             image_size_stg = 4G
             drive_format_stg = virtio
+            drive_format_image1 = ide
+            q35:
+                drive_format_image1 = ahci
```

Comment:
Another bug, not related with uninstall driver,  it is better to show this info in the commit message.

Reply:
We need ensure we use ide to start the guest or if we uninstall blk/scsi driver, it will BSOD. And certainly I can highlight it in commit info.

Reply:
Right, i know the purpose, my comment is only about explain this change in commit info, it is not related with uninstall driver, thanks!